pkg/serverconfig/types.go

```diff
@@ -0,0 +1,56 @@
+package serverconfig
+
```

Comment:
This is shared with the console operator at this point.  Ideally there would be 1 canonical reference, but for now we can at least separate it out into its own file & make it relatively easy to copy-pasta.  See [here](https://github.com/openshift/console-operator/blob/master/pkg/console/subresource/consoleserver/config.go).

Reply:
For history, we initially discussed moving to `openshift/api`, but its not API, has nothing to do with REST, and is not public.  Shot down.

Then we considered moving it to `library-go`, but that [PR was closed](https://github.com/openshift/library-go/pull/454) as nobody else wants our CLI config.  

Conclusion is that this is really fine for now, its no functionality, just an object defintion.